diff -rupN casacore-1.5.0/casa/aips.h casacore-1.5.0-new/casa/aips.h
--- casacore-1.5.0/casa/aips.h	2013-05-12 22:44:47.000000000 +0000
+++ casacore-1.5.0-new/casa/aips.h	2013-05-12 22:51:59.000000000 +0000
@@ -40,6 +40,8 @@
 //# Define the special aips++ macros
 #include <casa/aipsdef.h>
 
+#include <unistd.h>
+
 namespace casa { //# NAMESPACE CASA - BEGIN
 
 } //# NAMESPACE CASA - END
diff -rupN casacore-1.5.0/casa/System/Casarc.h casacore-1.5.0-new/casa/System/Casarc.h
--- casacore-1.5.0/casa/System/Casarc.h	2013-05-12 22:44:46.000000000 +0000
+++ casacore-1.5.0-new/casa/System/Casarc.h	2013-05-12 22:51:59.000000000 +0000
@@ -31,6 +31,7 @@
 #include <string>
 #include <list>
 #include <map>
+#include <unistd.h>
 
 namespace casa { //# NAMESPACE CASA - BEGIN
 
diff -rupN casacore-1.5.0/scimath/Mathematics/ConvolveGridder.tcc casacore-1.5.0-new/scimath/Mathematics/ConvolveGridder.tcc
--- casacore-1.5.0/scimath/Mathematics/ConvolveGridder.tcc	2013-05-12 22:44:40.000000000 +0000
+++ casacore-1.5.0-new/scimath/Mathematics/ConvolveGridder.tcc	2013-05-12 22:51:33.000000000 +0000
@@ -305,11 +305,11 @@ Bool ConvolveGridder<Domain, Range>::gri
 				    const Vector<Domain>& p,
 				    const Range& value)
 {
-  loc=location(loc,p);
+  loc=this->location(loc,p);
   loc-=offsetVec;
   if(onGrid(loc,supportVec)) {
     Bool del;
-    posVec=position(posVec, p);
+    posVec=this->position(posVec, p);
     const IPosition& fs = gridded.shape();
     vector<Int> s(fs.begin(), fs.end());
     switch(loc.nelements()) {
@@ -345,10 +345,10 @@ Bool ConvolveGridder<Domain, Range>::deg
 				    const Vector<Domain>& p,
 				    Range& value)
 {
-  loc=location(loc,p); 
+  loc=this->location(loc,p); 
  if(onGrid(loc,supportVec)) {
     Bool del;
-    posVec=position(posVec, p);
+    posVec=this->position(posVec, p);
     const IPosition& fs = gridded.shape();
     vector<Int> s(fs.begin(), fs.end());
     switch(loc.nelements()) {
